Configurations/10-main.conf

```diff
@@ -684,15 +693,15 @@ my %targets = (
         inherit_from     => [ "linux-generic64", asm("ppc64_asm") ],
         cflags           => add("-m64"),
         cxxflags         => add("-m64"),
-        cppflags         => add("-DB_ENDIAN"),
+        lib_cppflags     => add("-DB_ENDIAN"),
         perlasm_scheme   => "linux64",
         multilib         => "64",
     },
     "linux-ppc64le" => {
         inherit_from     => [ "linux-generic64", asm("ppc64_asm") ],
         cflags           => add("-m64"),
         cxxflags         => add("-m64"),
-        cppflags         => add("-DL_ENDIAN"),
+        lib_cppflags     => add("-DB_ENDIAN"),
```

Comment:
NOOOOOO! All of a sudden L_ENDIAN became B_ENDIAN...

Reply:
Oops, fixing